datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorConfig.java

```diff
@@ -110,6 +114,9 @@ public KafkaBasedConnectorConfig(Properties properties) {
         INCLUDE_DATASTREAM_NAME_IN_CONSUMER_CLIENT_ID, DEFAULT_INCLUDE_DATASTREAM_NAME_IN_CONSUMER_CLIENT_ID);
     _enablePartitionAssignment = verifiableProperties.getBoolean(ENABLE_PARTITION_ASSIGNMENT, Boolean.FALSE);
 
+    _callbackStatusStrategy = verifiableProperties.getString(CONFIG_CALLBACK_STATUS_STRATEGY_FACTORY_CLASS,
+        CallbackStatusWithComparableOffsetsFactory.class.getName());
+
```

Comment:
You can add verification as well here and save the object rather then saving the name and putting the onus on the caller to verify and throw exception.